src/System.Net.Security/src/System/Net/Security/TlsCipherSuiteData.Lookup.cs

```diff
@@ -0,0 +1,3395 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This file has been auto-generated. Do not edit by hand.
+// Instead open Developer Command prompt and run: TextTransform FileName.tt
+// Or set AllowTlsCipherSuiteGeneration=true and open VS and edit there directly
+
+// This line is needed so that file compiles both as a T4 template and C# file
+
+using System.Collections.Generic;
+using System.Security.Authentication;
+
+namespace System.Net.Security
+{
+    internal partial struct TlsCipherSuiteData
+    {
+        private const int LookupCount = 337;
```

Comment:
In cases like this, we'll often have a debug-only static cctor that validates the size, e.g.
```C#
#if DEBUG
static TlsCipherSuiteData()
{
    Debug.Assert(LookupCount == s_tlsLookup.Count, $"Expected {LookupCount} entries, got {s_tlsLookup.Count}");
}
```